src/components/status-panel/index.js

```diff
@@ -72,6 +73,16 @@ class StatusPanel extends Component {
 
 		const url = 'http://localhost:' + preferences.site.port;
 
+		async function openWPCLITerminal() {
+			await spawn( 'osascript', [ '-e', '"tell application \\"Terminal\\"\n ' +
+				' activate\n ' +
+				' set currentTab to do script (\\"alias wp=\'docker-compose run --rm cli \'\\")\n ' +
+				' delay 2\n ' +
+				' do script (\\"cd ' + preferences.basic[ 'gutenberg-folder' ] + ' \\") in currentTab\n ' +
```

Comment:
```suggestion
				' do script (\\"cd ' + preferences.basic[ 'wordpress-folder' ] + ' \\") in currentTab\n ' +
```

This looks like it should be `wordpress-folder`?

Reply:
this is gutenberg folder because that is where we have the cli docker container :)

Reply:
Oh, I see. That's the Gutenberg CLI docker container, not the TestPress CLI container. The latter is in `~/Library/Application Support/testpress/tools`.